libs/update_api_user_metrics.py

```diff
@@ -18,17 +20,47 @@
          count(*) AS "totalRequests",
          email
 FROM default.{table}
-GROUP BY email;
+GROUP BY email
 """
 
 
 ATHENA_BUCKET = "s3://covidactnow-athena-results"
+HUBSPOT_API_KEY = os.getenv("HUBSPOT_API_KEY")
 
 
 class CloudWatchQueryError(Exception):
     """Raised on a failed query to CloudWatch"""
 
 
+def update_hubspot_activity(email, latest_active_at, days_active):
+    """Updates Hubspot contact with latest activity dates."""
+    contacts_url = "https://api.hubapi.com/crm/v3/objects/contacts"
+
+    query_string = {"hapikey": HUBSPOT_API_KEY}
+
+    # Hubspot date field should be at UTC midnight.  Adding the timestamp parsing
+    # makes the date timestamp respect the UTC timezone.
```

Comment:
"Adding the timestamp parsing makes the date timestamp respect the UTC timezone" -> "When the %z directive is provided to the strptime() method, a TZ aware datetime object will be produced.".... Though I think I'd use 
date.isoformat() and datetime.combine(... tzinfo=datetime.timezone.utc).